### ISSUE TITLE
Random cursors on the right most column

### ISSUE THREAD

**Issue by N4tus** [zellij-org/zellij#489]
Inside neovim there is some flickering at the right most column whenever i press a key. To be axact the curser is rerenderd 1 to 3 times (seems random) additionally. (If I hold down a key, these flicker go crazy, which is quite distracting.) Since version 0.9.0 displays different cursers, the type of fake curser also changes.  
They are always on the right most column regardless of terminal size.

I using zellij in wsl2 with alacritty.

**Comment by imsnif** [comment]
That does sound quite distracting :) I know Zellij has performance issues on wsl2 (being actively worked on) - could this be related? Is Zellij particularly slow for you?

**Comment by N4tus** [comment]
>That does sound quite distracting :) I know Zellij has performance issues on wsl2 (being actively worked on) - could this be related? Is Zellij particularly slow for you?
Yes it is. As soon i maximaize alacritty, i notice input lag (also the flickering is more intense). With a small window the lag is still there, but minimal enough to not bother me.
